acm/src/main/java/org/literacybridge/acm/utils/DropboxFinder.java

```diff
@@ -0,0 +1,129 @@
+package org.literacybridge.acm.utils;
+
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.map.ObjectMapper;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+
+/**
+ * Created by bill on 2/24/16.
+ */
+public class DropboxFinder {
```

Comment:
Out of curiosity: Does the Dropbox Java SDK maybe provide similar functionality?